oleesir/Banking-app
# #164726367User should be able to see the bank account profile (dashboard).

-implement profile-template

[Finishes #164726367]

#### What does this PR do?
implement account-profile-template
#### Description of Task to be completed?
implement account-profile-template
#### How should this be manually tested?
* Navigate to the UI folder and click on profile.html in your browser
* You will see an account-profile-template
#### What are the relevant pivotal tracker stories?
[#164726367](https://www.pivotaltracker.com/story/show/164726367)
#### Any background context you want to add?
N/A
#### Screenshots
<img width="1280" alt="Screenshot 2019-03-25 at 9 59 01 AM" src="https://user-images.githubusercontent.com/17679511/54906957-b6ece800-4ee4-11e9-8433-2282444b8568.png">
